patches/third_party-blink-renderer-core-dom-document.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/third_party/blink/renderer/core/dom/document.cc b/third_party/blink/renderer/core/dom/document.cc
+index c0ade72c75bd1c0212531b4adcd5386d4597af8f..513545e0968623803fe4a1b7222b6a521e0df631 100644
+--- a/third_party/blink/renderer/core/dom/document.cc
++++ b/third_party/blink/renderer/core/dom/document.cc
+@@ -33,7 +33,7 @@
+ // instead of including more headers. If that is infeasible, adjust the limit.
+ // For more info, see
+ // https://chromium.googlesource.com/chromium/src/+/HEAD/docs/wmax_tokens.md
+-#pragma clang max_tokens_here 900000
++#pragma clang max_tokens_here 2000000
```

Comment:
I don't understand why we are changing this. I think we should keep the upstream value unless we have a very good reason to change it

Reply:
I think this change should be reverted. I did a release and component linux build with no issues

Reply:
actually I did hit this in component build so I guess we'll leave it for now